sample/src/main/java/org/smartregister/sample/view/SampleReportModule.java

```diff
@@ -0,0 +1,47 @@
+package org.smartregister.sample.view;
+
+import android.view.ViewGroup;
+
+import org.smartregister.reporting.domain.IndicatorTally;
+import org.smartregister.reporting.impl.ReportingModule;
+import org.smartregister.reporting.impl.models.IndicatorModel;
+import org.smartregister.reporting.impl.views.NumericIndicatorView;
+import org.smartregister.reporting.impl.views.PieChartIndicatorView;
+import org.smartregister.sample.R;
+import org.smartregister.sample.utils.ChartUtil;
+
+import java.util.List;
+import java.util.Map;
+
+import static org.smartregister.reporting.impl.ReportingUtil.getIndicatorModel;
+import static org.smartregister.reporting.impl.ReportingUtil.getPieChartViewModel;
+import static org.smartregister.reporting.impl.views.IndicatorView.CountType.LATEST_COUNT;
+import static org.smartregister.reporting.impl.views.IndicatorView.CountType.STATIC_COUNT;
+import static org.smartregister.reporting.impl.views.IndicatorViewFactory.createView;
+
+public class SampleReportModule implements ReportingModule {
```

Comment:
Is the idea to have different reporting modules with different views? 
Won't they usually be displayed in one dashboard?

Reply:
You are right they will be displayed in same view hierarchy in the dashboard. A case where we have 100 numeric indicators and pie chart, having too many views for different indicators is less maintainable, that's the reason for separating the implementations.